src/commands/env/create/compute.ts

```diff
@@ -64,9 +63,7 @@ export default class EnvCreateCompute extends Command {
           )} to the "features" list in your scratch org definition JSON file, e.g. "features": ["Functions"]`
       );
     }
-    if (!flags.json) {
-      cli.action.start(`Creating compute environment for org ID ${orgId}`);
-    }
```

Comment:
Did you QA this?
I'm not sure that `cli.action.start()` has the same `--json` checks that `this.log()` does.

Reply:
Actions go out on stderr, so this shouldn't matter.

Reply:
_"He says 99% sure."_

Reply:
https://github.com/oclif/cli-ux/blob/21e8525497e6b4e9b4fa6cc1e2006bae2ff402eb/src/action/base.ts#L29